modules/jdbc/src/integrationTest/java/org/apache/ignite/jdbc/ItJdbcClientTimeZoneTest.java

```diff
@@ -176,25 +180,98 @@ public void dynamicParamRespectsTimeZone() throws SQLException {
                     // Client time zone is "GMT".
```

Comment:
considering the very first lines of this method, "session" time zone refers to time zone of the connection, while "client" tz refers to time zone of the JVM. Let's change all comments accordingly, because after your changes they are no longer correct in certain places.